spinflip/diagonalizer.py

```diff
@@ -136,7 +152,7 @@ def multi_H_Plotter(H_array, t_lim_array = 'timescale', quantity_array = np.arra
 
     for n in np.arange(0, N):
         for k in quantity_array[n]:
-            ax[0,n].plot(np.linspace(0,t_lim_array[n],resolution), state_vs_time_array[n,:,k,k], label = 
+            ax[0,n].plot(np.linspace(0,t_lim_array[n],resolution)*1000, state_vs_time_array[n,:,k,k], label = 
```

Comment:
Why was this multiplied by 1000 in this version?

Reply:
x axis is in ms. not sure at what point I changed the label to ms but going through the units that factor is correct to set the values to ms.